### PR TITLE
Add paraview

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/paraview/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - twisted
+    - autobahn
+    - six
+    - protobuf
+    - lz4-c
+    - zope.interface
+    - expat
+    - libogg
+    - libtheora
+    - libnetcdf
```
Comment (ocefpaf) [inline recipes/paraview/meta.yaml]: We need to pin this one too.

File: recipes/paraview/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+test:
+  imports:
+    - paraview
+  commands:
+    - pvpython --version 2>&1 | grep paraview
+
+about:
+   home: http://www.paraview.org
+   license: BSD
```
Comment (ocefpaf) [inline recipes/paraview/meta.yaml]: Do you know the version? `BSD 2-Clause` or `BSD 3-Clause`?

File: recipes/paraview/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipes/paraview/meta.yaml]: ok

File: recipes/paraview/meta.yaml (hunk 2 above)
Comment (jschueller) [inline recipes/paraview/meta.yaml]: 3-Clause

**Comment by jschueller** [comment]
cc @bgruening

**Comment by jschueller** [comment]
@ocefpaf this is ready

File: recipes/paraview/build.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+rm -r ./Plugins/*
```
Comment (ocefpaf) [inline recipes/paraview/build.sh]: Can you add a comment why we need to remove this directory?

File: recipes/paraview/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  version: {{ version }}
+
+source:
+  fn: ParaView-v{{ version }}.tar.gz
+  url: http://paraview.org/files/v{{ maj_min }}/ParaView-v{{ version }}.tar.gz
+  sha256: f488d84a53b1286d2ee1967e386626c8ad05a6fe4e6cbdaa8d5e042f519f94a9
+
+build:
+  number: 0
+  # only enough time on circleci
```
Comment (ocefpaf) [inline recipes/paraview/meta.yaml]: :-(

We can try manual uploads for OS X

**Pull request merged**